src/ModRamPlasmasphere.f90

```diff
@@ -61,6 +65,32 @@ subroutine plasmasphere(dt)
               if (dts >= dt) exit
            enddo
 
+        case('Vania')
+            ! Only works if NR=20, NT=49
```

Comment:
Actually, can we put in a check for this? I'm not sure what the failure mode looks like but it seems unlikely that any error messages or tracebacks will be particularly intelligible.

Reply:
I'm actually thinking we should just pull this from the repo. It was originally included because we were having troubles with the other plasmasphere things and wanted to compare with the older method. We could include it in the future (since it does use the flux corrected transport model) as another option. But I would want to make it robust to grid changes if we did that which would take a little more work than I can invest right now.